build/deps.mk

```diff
@@ -3,37 +3,30 @@
 #
 
 GO           ?= go
+GOFMT        ?= gofmt
 VENDOR_CMD   ?= ${GO} mod tidy
 BUILD_DIR    ?= ./bin/
 
-# These should be mirrored in /tools.go to keep versions consistent
-# GOTOOLS      += github.com/client9/misspell/cmd/misspell
+# Go file to track tool deps with go modules
+TOOL_DIR     ?= tools
+TOOL_CONFIG  ?= $(TOOL_DIR)/tools.go
 
+GOTOOLS ?= $(shell cd $(TOOL_DIR) && go list -f '{{ .Imports }}' -tags tools |tr -d '[]')
```

Comment:
Neet trick, any imports defined in this package will become tools to install. No more need to `+=` them constantly

Reply:
Yeah, I like that.